data/events/2024/birmingham-al/main.yml

```
@@ -104,10 +100,16 @@ sponsors:
     level: silver
   - id: assessivate
     level: silver
-  # - id: sparkhound
-  #   level: silver
-  # - id: itac
-  #   level: bronze
+  - id: sparkhound
+    level: silver
+  - id: itac
```

There is no sponsor profile called `itac`. Did you mean `itac-solutions`? If so:
```suggestion
  - id: itac-solutions
```

If not, please create a new sponsor profile for this sponsor.

---

Great catch, yes, itac-solutions was the correct sponsor. I've updated the branch